packages/celotool/src/cmds/account/faucet.ts

```diff
@@ -42,19 +46,38 @@ export const builder = (argv: yargs.Argv) => {
         })
       },
     })
-    .option('dollar', {
-      type: 'number',
-      description: 'Number of dollars to faucet',
-      demand: 'Please specify dollars to faucet',
-    })
-    .option('gold', {
-      type: 'number',
-      description: 'Amount of gold to faucet',
-      demand: 'Please specify gold to faucet',
+    .array('tokenParams')
+    .option('tokenParams', {
+      type: 'string',
+      description: '<token,amount> pair to faucet',
+      demand:
+        'Please specify stableToken,amount pairs to faucet (ex: --tokenParams CELO,3 cUSD,10 cEUR,5)',
+      coerce: (pairs) => {
+        // Ensure that pairs are formatted properly and use possible tokenParams
+        const validCeloTokens = Object.values(celoTokenInfos).map((tokenInfo) => {
```

Comment:
The celoTokenInfos, has all the possible tokens (stables and celo), but, it does not check if it has the contract deployed or not. This other function `validCeloTokenInfos()`, returns exactly the same, but at least checks it the contract exists in the chain.
It's your call here, we didn't defined which behaviour we should have. In your case, that transaction will fail, using this validCeloToken, will fail at the checking level, and not in the middle of the tx

Reply:
that's a great point! i think only downside with `validCeloTokenInfos` is that it needs to happen after we instantiate the `kit` (so within the main fauceting func instead of arg processing). Since the use case for this should pretty much only be for alfajores (maybe baklava) these days, I'm leaning towards sticking with the earlier/faster fail as it currently is, as opposed to the more thorough check with `validCeloTokenInfos`, wdyt?

Reply:
actually just decided to add this as an option that folks can choose to turn on if debugging :)

Reply:
Great call